training/general/src/environments/sklearn-1.1-ubuntu20.04-py38-cpu/context/intel-extension/conda_dependencies.yaml

```diff
@@ -0,0 +1,22 @@
+name: sklearn-1.1
+channels:
+- conda-forge
+- anaconda
+dependencies:
+- python=3.9
+- pip=21.3.1
+- pandas~=1.3.0
+- scipy~=1.7.0
+- numpy~=1.21.0
+- scikit-learn-intelex~={{latest-sklearnex-version}}
```

Comment:
what is the size of intel extensions?

Reply:
The size of the intel extension docker image  is 2.68GB and that of the stock version is 2.89 GB